src/OrbitBase/include/OrbitBase/TypedefUtils.h

```diff
@@ -15,6 +16,39 @@ using EnableIfUConvertibleToT = std::enable_if_t<std::is_convertible_v<U, T>>;
 template <typename T, typename U>
 using EnableIfUNotConvertibleToT = std::enable_if_t<!std::is_convertible_v<U, T>>;
 
+template <typename OtherSummandTag>
+struct PlusTagBase {};
+
+struct DefaultPlus {
+  template <typename T, typename U>
+  auto operator()(T&& t, U&& u) const {
+    return std::forward<T>(t) + std::forward<U>(u);
+  }
+};
+
+constexpr DefaultPlus kDefaultPlus{};
+
+struct DefaultMinus {
+  template <typename T, typename U>
+  auto operator()(T&& t, U&& u) const {
+    return std::forward<T>(t) - std::forward<U>(u);
+  }
+};
+
+constexpr DefaultMinus kDefaultMinus{};
+
+struct DefaultTimes {
+  template <typename T, typename U>
+  auto operator()(T&& t, U&& u) const {
+    return std::forward<T>(t) * std::forward<U>(u);
+  }
+};
+
+constexpr DefaultTimes kDefaultTimes;
```

Comment:
nit: Maybe `{} for consistency?

Reply:
Good point. But I'll rather remove `{}` in the other two constant definitions.